src/test/java/org/example/tests/ReopenAppLogoutTest.java

```diff
@@ -0,0 +1,32 @@
+package org.example.tests;
+
+import io.appium.java_client.android.AndroidDriver;
+import org.example.enums.UserCredentials;
+import org.example.pages.LoginPageBase;
+import org.example.pages.ProductPageBase;
+import org.example.utils.BaseTest;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ReopenAppLogoutTest extends BaseTest {
+
+    @Test
+    public void testAppReopenAfterLogin() {
+        LoginPageBase loginPage = initPage(getDriver(), LoginPageBase.class);
+        ProductPageBase productPage = loginPage.login(UserCredentials.STANDARD_USER);
+        Assert.assertTrue(productPage.isProductListVisible(), "Product list not visible after login.");
+
+        WebDriver driver = getDriver();
+
+        if (driver instanceof AndroidDriver) {
+            ((AndroidDriver) driver).closeApp(); //simulate close
```

Comment:
Carina has the IMobileUtils interface, you can leverage it to handle closing and launching the app, rewrite it with CArina